### PR TITLE
Add basic acceptance test

### PR DESCRIPTION
**Pull request by ehelms**
This is *very* basic at the moment in terms of what it verifies. The biggest win I am aiming for here is that PRs will get automated testing of the module and help speed up fixes for developers.

### CODE REVIEW

**Comment by ehelms** [comment]
@jturel you might be interested in this as it could speed up test cycles

**Comment by jturel** [comment]
I like it. Seems like the new tests weren't run based on the github action logs, or maybe I missed it?

File: spec/setup_acceptance_node.pp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+# Necessary for PostgreSQL EVR extension
+yumrepo { "pulpcore":
+  baseurl  => "http://yum.theforeman.org/pulpcore/3.14/el\$releasever/x86_64/",
+  descr    => "Pulpcore",
+  enabled  => true,
+  gpgcheck => true,
+  gpgkey   => "https://yum.theforeman.org/pulpcore/3.14/GPG-RPM-KEY-pulpcore",
+}
```
Comment (ekohl) [inline spec/setup_acceptance_node.pp:23]: Not just for EVR, also Pulp itself. However, we should have the Pulpcore module here
```suggestion
class { 'pulpcore::repo':
}
```

**Comment by ehelms** [comment]
> I like it. Seems like the new tests weren't run based on the github action logs, or maybe I missed it?

Good catch, I forget that our modulesync only adds acceptance tests to the github workflow if there are actual acceptance tests. I've added that now.

File: spec/setup_acceptance_node.pp (hunk 1 above)
Comment (ehelms) [reply spec/setup_acceptance_node.pp:23]: We should, if this module actually did anything with Pulpcore :) This is similar to the puppet-katello acceptance tests, this does not rely on puppet-foreman_proxy_content which handles setting up Pulp 3. And for the installer, the answer file brings the modules together. This module has no knowledge of getting Pulp set up.

Comment (ekohl) [reply spec/setup_acceptance_node.pp:23]: Ah right

**Review by ekohl — APPROVED** [review]

**Pull request merged**